src_new/adapters/prebid.js

```diff
@@ -617,12 +617,18 @@ function fetchBids(activeSlots, impressionID){
 				};
 
 				if (CONFIG.getGdpr()) {
-					prebidConfig["consentManagement"] = {
+					prebidConfig["consentManagement"]["gdpr"] = {
```

Comment:
prebidConfig["consentManagement"] is not assigned a value as an empty object.
isn't it throwing some error?

Reply:
Fixed the issue. Thanks :)